instructors.md

```diff
@@ -74,13 +74,20 @@ as long as learners using Windows do not run into roadblocks such as:
 
 *   Time estimates:
     *   @gvwilson: 3 hours
-
+	
+*	How to use the materials in the shell-novice repository (or, lesson planning)
+	* Use the `filesystem` directory for in-workshop exercises and live coding examples.  
+	The easiest way to get can get these files is by cloning the entire shell-novice directory.  Alternatives include creating a zipped copy of the `filesystem` directory and making that available to learners.  
+	* Website: Can give links to learners so they can follow along and see exercises (particularly if you're following the lesson content without many changes).  Make sure to point to website as a post-workshop reference.  
```

Comment:
On Thu, Apr 09, 2015 at 01:12:49PM -0700, Christina Koch wrote:
> > + * Website: Can give links to learners so they can follow along
> >     and see exercises (particularly if you're following the lesson
> >     content without many changes).  Make sure to point to website
> >     as a post-workshop reference.
> 
> … So I have mixed feelings but wanted to be fair to the input of
> other people.

That sounds reasonable to me.  If we want to put forward both
positions and not make a firm suggestion, I think we should be laying
out the arguments in more detail.  I'm not sure if Greg's suggestions
are echoed somewhere in the repository's notes, but the “don't show
these live” recommendation is in the “What the Lesson Notes Are For”
of [1].

[1]: http://software-carpentry.org/blog/2015/03/teaching-tips.html